osidb/models.py

```diff
@@ -2851,6 +2866,13 @@ def create_flawmeta(flaw, _type, meta, **extra_fields):
             )
 
 
+@pghistory.track(
+    pghistory.InsertEvent(),
+    pghistory.UpdateEvent(),
+    pghistory.DeleteEvent(),
+    exclude="meta_attr,_alerts",
+    model_name="FlawMetaAudit",
+)
```

Comment:
`FlawMeta` is a relic model and a subject for a removal (with the next major release). All of its parts were already migrated into other places (like `FlawReference` etc.). You can simply exclude it from the audit.